navigation.yaml

```diff
@@ -1158,19 +1162,19 @@ pricing:
 
 documentation:
   title: Docs
-  path: /_docs
+  path: /0_docs
```

Comment:
Why change this url?

Reply:
_docs has an underscore and toto and robin stopped underscore docs from being rendered by flask